Makefile

```diff
@@ -12,9 +16,9 @@ ifneq ($(RELEASE_VERSION),)
 	VERSIONLDFLAG := -X github.com/tektoncd/cli/pkg/cmd/version.clientVersion=$(RELEASE_VERSION)
 endif
 
-FLAGS := $(VERSIONLDFLAG)$(SKIPLDFLAG)
+FLAGS := $(VERSIONLDFLAG)$(SKIPLDFLAG)$(NAMESPACELDFLAG)
 ifneq ($(SKIPLDFLAG),)
-	FLAGS := $(VERSIONLDFLAG) $(SKIPLDFLAG)
+	FLAGS := $(VERSIONLDFLAG) $(SKIPLDFLAG) $(NAMESPACELDFLAG)
```

Comment:
we need to handle the cases here carefully, I think it will cause an issue when you will have skip flag but not namespace flag

or 

we need to handle namespace flag case

Reply:
I have tested without specifying `skip` flag like below

```
GOOS=linux GOARCH=amd64 go build -mod=vendor -ldflags "-X github.com/tektoncd/cli/pkg/cmd/version.namespace=tekton-pipelines"
```

and it works without any issue 

let me know if i miss understand the point which you have mentioned

Reply:
You are not using makefile here :D

Reply:
Updated makefile

tested with following scenario

* `NAMESPACE=tekton-pipelines SKIP_CHECK_FLAG=true RELEASE_VERSION=0.13 make amd64`
* `NAMESPACE=tekton-pipelines make amd64`
* `NAMESPACE=tekton-pipelines RELEASE_VERSION=0.13 make amd64`
* `SKIP_CHECK_FLAG=true NAMESPACE=tekton-pipelines make amd64`